### DISCUSSION TITLE
why ？API not response

### DISCUSSION THREAD

**Discussion by KevinShen-zz** [xtekky/gpt4free#480] [Q&A]
Unable to fetch the response, Please try again.
<img width="981" alt="image" src="https://user-images.githubusercontent.com/990429/236596787-bb9b1b00-11ce-4774-86b3-4425d1ad31bc.png">

**Comment by xiangsx** [comment]
> Unable to fetch the response, Please try again.
> <img width="981" alt="image" src="https://user-images.githubusercontent.com/990429/236596787-bb9b1b00-11ce-4774-86b3-4425d1ad31bc.png">
> 

look this issue
#484

**Comment by unknownparticles** [comment]
I encountered the same problem.

**Comment by lunhui1124500398** [comment]
I also encountered the same problem.

**Comment by sergioparadiso** [comment]
change your client in /gpt4free/you
` 'user-agent': 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/111.0.5563.110 Safari/537.36',
`
i'm using windows 11, py 3.11, doesnt work on ubuntu server though  (also using a linux agent doesn't fix)
